src/Process/3dTilesProcessing.js

```diff
@@ -144,8 +144,8 @@ export function process3dTilesNode(cullingTest, subdivisionTest) {
         if (node.parent.pendingSubdivision && !node.parent.additiveRefinement) {
             node.visible = false;
             // TODO: node.setDisplayed(false)?
-            if (node.material) {
-                node.material.visible = false;
+            if (node.content) {
```

Comment:
I created a setDisplayed function.

I also simplified the code a bit by removing the calls to setDisplayed when the tile was not visible.